packages/app/pages/EconomicEvent/EconomicEvent.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from "react";
```

Comment:
Please add a brief header to your file in javadoc style

Reply:
Added a header, matching yours in format
:+1: